### ISSUE TITLE
Sphincs implementation version dubious

### ISSUE THREAD

**Issue by baentsch** [open-quantum-safe/liboqs#1475]
While implementing https://github.com/open-quantum-safe/oqs-provider/issues/157 it became obvious (due to no difference in version IDs) that https://github.com/open-quantum-safe/liboqs/pull/1420 did not update the version name/ID for the new code version. This is somewhat confusing when listing old and new OIDs: See https://github.com/open-quantum-safe/oqs-provider/blob/mb-algdocs/oqs-template/oqs-sig-info.md : All Sphincs+ algorithms are listed under the same moniker/version string.

As the Sphincs algorithm naming got a complete revamp with https://github.com/open-quantum-safe/liboqs/pull/1467, it is no real problem, but this issue is to ask whether it is intentional that there is no (different) version name between the previous Sphincs code base and the current one.

**Comment by dstebila** [comment]
You're right, I see that the `spec-version` line in https://github.com/open-quantum-safe/liboqs/pull/1420/files#diff-cfe013791f73d0afd5867c1e07d9d2e33112f0e04dae3d989fdd67a98234a19fR26 was not updated. I've fixed this is #1475, does that address the downstream issue?
